src/services/firebase.ts

```diff
@@ -92,6 +92,22 @@ export namespace ourFirebase {
       .on('value', snap => getMatchMemberships(snap ? snap.val() : {}));
   }
 
+  export function addFcmToken(fcmToken: string, platform: 'ios' | 'android') {
+    checkFunctionIsCalledOnce('addFcmToken');
+    const user = assertLoggedIn();
+    const fcmTokenObj: fbr.FcmToken = {
+      lastTimeReceived: <any>firebase.database.ServerValue.TIMESTAMP,
+      platform: platform
+    };
+    return db()
+      .ref(
+        `gamePortal/gamePortalUsers${
```

Comment:
you forgot a '/' :
gamePortal/gamePortalUsers
to:
gamePortal/gamePortalUsers/